tls/extensions/s2n_server_max_fragment_length.c

```diff
@@ -77,7 +77,7 @@ static int s2n_max_fragment_length_recv(struct s2n_connection *conn, struct s2n_
      *# larger than the negotiated length is sent.
      */
     conn->negotiated_mfl_code = mfl_code;
-    conn->max_outgoing_fragment_length = MIN(conn->max_outgoing_fragment_length, mfl_code_to_length[mfl_code]);
+    POSIX_GUARD_RESULT(s2n_connection_set_max_fragment_length(conn, conn->max_outgoing_fragment_length));
```

Comment:
Kind of confusing bc you're setting the max fragment length with conn->max_outgoing_fragment_length. I'm sure it's right, it's just weird to read.

Reply:
I could set it to MIN(conn->max_outgoing_fragment_length, mfl_code_to_length[mfl_code]) but that felt repetitive since s2n_connection_set_max_fragment_length enforces the mfl_code anyway :\